JoyfulPrototype/Assets/Scripts/PlayerController.cs

```diff
@@ -62,6 +62,9 @@ public class PlayerController : MonoBehaviour {
     private float _shotDelayCounter;
     private float _timeFalling;
 
+    public bool JumpPowerUp { get; set; } 
+    private int JumpMod = 1;

```

Comment:
@Renigma , Just FYI.
I added the two missing variables above, I'm not sure what the intended value for jump Mod should be so I set it to 1. 

-Angel